packages/logging_cloudwatch/aws_logging_cloudwatch/lib/src/cloudwatch_logger_plugin.dart

```diff
@@ -169,19 +183,20 @@ class CloudWatchLoggerPlugin extends AWSLoggerPlugin
 
     if (!_syncing) {
       _syncing = true;
+
       DateTime? nextRetry;
+
       try {
         await startSyncing();
       } on _TooNewLogEventException catch (e) {
-        nextRetry =
-            DateTime.fromMillisecondsSinceEpoch(e.timeInMillisecondsSinceEpoch)
-                .add(_minusMaxLogEventTimeInFuture);
+        nextRetry = DateTime.fromMillisecondsSinceEpoch(
+          e.timeInMillisecondsSinceEpoch,
+        ).add(_minusMaxLogEventTimeInFuture);
       } on Exception catch (e) {
         logger.error('Failed to sync logs to CloudWatch.', e);
       } finally {
-        _handleFullLogStoreAfterSync(
-          retryTime: nextRetry,
-        );
+        _handleFullLogStoreAfterSync(retryTime: nextRetry);
```

Comment:
Do you have autolint setup?  I'm a bit confused by the inconsistent changes here. 

In the above line, you added a "," to a single parameter method call, but then below you remove a "," in a single parameter method call.